analyzer/runtime/extract.go

```diff
@@ -1037,6 +1056,69 @@ func extractEvents(blockData *BlockData, relatedAccountAddresses map[apiTypes.Ad
 					}
 					return nil
 				},
+				IUniswapV2FactoryPairCreated: func(token0ECAddr ethCommon.Address, token1ECAddr ethCommon.Address, pairECAddr ethCommon.Address, allPairsLength *big.Int) error {
```

Comment:
Is the plan to populate the `eventData.{RelatedAddresses, EvmLog{Name,Signature,Params}}` in this PR? I imagine we'll want to show this info in the explorer eventually

Reply:
oh have we been doing EvmLog ones?

related addresses we should do

Reply:
oh I see about the evm log stuff. I'll try doing it similar to the surrounding code

Reply:
added

Reply:
gonna resolve